pkgs/development/python-modules/pysingleton/default.nix

```
@@ -0,0 +1,23 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+}:
+
+buildPythonPackage rec {
+  pname = "pysingleton";
+  version = "0.2.1";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "5776e7a4ba0bab26709da604f4e648c5814385fef34010723db3da0d41b0dbcc";
+  };
+
+  # No tests in the Pypi package.
+  doCheck = false;
+
+  meta = with lib; {
+    description = "Module which provides a decorator to create thread-safe singleton classes ";
```

I'm getting wrapping
```suggestion
    description = "Provides a decorator to create thread-safe singleton classes";
```